### PR TITLE
More Logging Info

### PR DESCRIPTION
**Pull request by Maithem**
Added more logging info to the checkpoint stats log.

### CODE REVIEW

File: runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java [hunk 1]
```diff
@@ -105,8 +105,8 @@ public long appendCheckpoints(CorfuRuntime rt, String author,
                 }
                 final long mapCpEnd = System.currentTimeMillis();
 
-                log.info("appendCheckpoints: took {} ms to checkpoint map {}",
-                        mapCpEnd - mapCpStart, streamId);
+                log.info("appendCheckpoints: took {} ms to checkpoint {} entries for map {}",
+                        mapCpEnd - mapCpStart, ((SMRMap) map).size(), streamId);
```
Comment (corfu-repo) [inline runtime/src/main/java/org/corfudb/runtime/MultiCheckpointWriter.java]: MINOR Remove this use of "SMRMap"; it is deprecated. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

**Comment by corfu-repo** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **e534c1a**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #906 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/AanqohTNdsL1T7h5fp4XuScMNYgAmGog)

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **ef8f25d**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Iterator workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #906 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/bQEumGADGx8AvZspYy4H7S0hGDH8jTdl)

**Pull request closed without merging**